packages/react-components/react-carousel-preview/src/components/Carousel/useCarousel.ts

```diff
@@ -133,6 +149,8 @@ export function useCarousel_unstable(props: CarouselProps, ref: React.Ref<HTMLDi
     root: slot.always(
       getIntrinsicElementProps('div', {
         ref: useMergedRefs(ref, carouselRef, rootRef),
+        role: 'region',
+        'aria-roledescription': 'carousel',
```

Comment:
aria-roledescription strings need to be localized, I think we can put this in the doc examples, but probably not in internal code